### PR TITLE
Update test and documentation

### PR DESCRIPTION
**Pull request by MM3103**
Update test and documentation

### CODE REVIEW

File: src/main/resources/application-dev.yaml [hunk 1]
```diff
@@ -4,9 +4,11 @@ spring:
...
   liquibase:
     enabled: true
-    change-log: ${LIQUIBASE_CHANGE-LOG:classpath:h2/changelogs.yaml}
+    change-log: ${LIQUIBASE_CHANGE-LOG:classpath:db/h2/changelogs.yaml}
     contexts: dev
   application:
     name: ${APPLICATION_NAME:orderService}
   main:
     allow-circular-references: true
+my:
```
Comment (AntonPozdeev) [inline src/main/resources/application-dev.yaml:13]: is it correct path?

File: src/test/java/my/pr/service/CityServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        assertEquals(3, cityService.getAllCities().size());
+    }
+
+    @Test
+    public void getAllCitiesTest() {
+        cityService.addCity(city1);
+        cityService.addCity(city2);
+        cityService.addCity(city3);
+        assertEquals(3, cityService.getAllCities().size());
+        assertEquals(city1.getName(), cityService.getAllCities().get(0).getName());
```
Comment (AntonPozdeev) [inline src/test/java/my/pr/service/CityServiceTest.java:53]: order can be another, therefore test can fail, try to get city by name or id

File: src/test/java/my/pr/service/StreetServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+    @Before
+    public void setUp() throws Exception {
+        cityRepository.save(city1);
+        street1.setCity(city1);
+        street2.setCity(city1);
+        street3.setCity(city1);
+    }
+
+    @After
+    public void tearDown() {
```
Comment (AntonPozdeev) [inline src/test/java/my/pr/service/StreetServiceTest.java:50]: need to delete city1 also

File: src/test/java/my/pr/service/StreetServiceTest.java [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+        assertEquals(3, streetService.getAllStreets().size());
+    }
+
+    @Test
+    public void getAllStreetsTest() {
+        streetService.addStreet(street1);
+        streetService.addStreet(street2);
+        streetService.addStreet(street3);
+        assertEquals(3, streetService.getAllStreets().size());
+        assertEquals(street1.getName(), streetService.getAllStreets().get(0).getName());
```
Comment (AntonPozdeev) [inline src/test/java/my/pr/service/StreetServiceTest.java:68]: order can be another, therefore test can fail, try to get street by name or id

File: src/main/resources/application-dev.yaml (hunk 1 above)
Comment (MM3103) [reply src/main/resources/application-dev.yaml:13]: Yes

Comment (MM3103) [reply src/main/resources/application-dev.yaml]: fixed

File: src/test/java/my/pr/service/CityServiceTest.java (hunk 2 above)
Comment (MM3103) [reply src/test/java/my/pr/service/CityServiceTest.java]: fixed

File: src/test/java/my/pr/service/StreetServiceTest.java (hunk 3 above)
Comment (MM3103) [reply src/test/java/my/pr/service/StreetServiceTest.java:50]: fixed

File: src/test/java/my/pr/service/StreetServiceTest.java (hunk 4 above)
Comment (MM3103) [reply src/test/java/my/pr/service/StreetServiceTest.java]: fixed

File: documentation/administration-guide/README.md [hunk 5]
```diff
@@ -8,14 +8,17 @@ Order service is a service for manages orders for the provision of various servi
 
 When a new order is created, it receives the status WAITING. 
 After the order status can be changed to ACCEPTED or REJECTED, depending on the decision of the administrator. 
-When an order has the status ACCEPTED or REJECTED its status can no longer be changed
+When an order has the status ACCEPTED or REJECTED its status can no longer be changed.
+At the end of the day, accepted and rejected orders receive the status ARCHIVED.
```
[the remark this answers is not in the document]
Comment (MM3103) [reply documentation/administration-guide/README.md]: fixed

File: documentation/administration-guide/README.md [hunk 6]
```diff
@@ -8,14 +8,17 @@ Order service is a service for manages orders for the provision of various servi
...
 
 ### Administrator
 
 Administrator manages the life cycle of all orders. 
+The administrator creates a list of cities and streets where the user can place orders. 
 When a user places an order, a message with the number of this order and a link is sent to the admin's mail. 
 The task of the admin is to review the order and either accept or reject it. 
 Depending on this, the order status will be changed to ACCEPTED or REJECTED. 
+Admin adds cities and streets where users can create orders
```
[the remark this answers is not in the document]
Comment (MM3103) [reply documentation/administration-guide/README.md]: fixed

File: documentation/development-guide/README.md [hunk 7]
```diff
@@ -1,19 +1,31 @@
 # ORDER SERVICE 
 Order service is a service for creation orders for the provision of various services 
 ##Order model
-Each order consists of seven fields: id, first name, last name, email, type of order, address and order status. 
+Each order consists of fourteen fields: id, first name, last name, email, type of order,city_id,street_id,house,flat, address,
```
[the remark this answers is not in the document]
Comment (MM3103) [reply documentation/development-guide/README.md]: fixed

File: documentation/development-guide/README.md [hunk 8]
```diff
@@ -1,19 +1,31 @@
...
+The table orders consists of fourteen fields: id, first name, last name, email, type of order,city_id,street_id,house,flat, address,
+creation time ,modification time, self installation and order status.  
+The table cities consists of two fields: id and name.
+The table streets consists of three fields: id, name and city_id.
 To deploy the database, the application uses liquid base.
 ##Order service
 The order service implements various operations, such as: adding a new order, deleting an order, updating an order, reading an order by ID. 
 In addition, it implements special methods for the admin, such as: reading all orders, accepting and rejecting an order.
+##City service
+The city service implements various operations, such as: adding a new city, deleting an city, updating an city, reading an city by ID.
```
[the remark this answers is not in the document]
Comment (MM3103) [reply documentation/development-guide/README.md]: fixed

File: documentation/installation-guide/README.md [hunk 9]
```diff
@@ -84,6 +84,5 @@ This section describes the minimum and recommended available resources required
```
[the remark this answers is not in the document]
Comment (MM3103) [reply documentation/installation-guide/README.md:88]: fixed

**Pull request merged**